packages/frontend/next.config.js

```diff
@@ -5,6 +5,10 @@ const nextConfig = {
   experimental: {
     optimizePackageImports: ["react", "react-dom"],
   },
+  // For Temporary S3 ENDPOINT URL
+  env: {
+    S3_PUBLIC_URL: process.env.S3_PUBLIC_URL || "",
```

Comment:
For nextjs, if you want to use environment variable, we need to specify the prefix, `NEXT_PUBLIC_`